pkg/controller/installation/products/rhsso/reconciler.go

```diff
@@ -107,6 +116,20 @@ func (r *Reconciler) Reconcile(ctx context.Context, inst *v1alpha1.Installation,
 		return phase, err
 	}
 
+	phase, err = r.ReconcileOauthClient(ctx, inst, &oauthv1.OAuthClient{
```

Comment:
I would leave this as part of the `setupOpenshiftIDP` function if it is all related to openshift idp setup